.eslintrc.json

```diff
@@ -33,7 +33,15 @@
     "before": true,
     "beforeEach": true,
     "after": true,
-    "afterEach": true
+    "afterEach": true,
+    "chai": true,
```

Comment:
Does chai itself definitely need to be available globally?

Reply:
Yes, chai is being used in `swaggerspec` `lisk_core` and `lisk_request_limit`

Reply:
1. `test/common/swaggerSpec.js` uses Chai to extend it with a custom assertion. This could be moved to the setup (so that assertion could then be used in other tests if needed).
1. `test/unit/api/fittings/lisk_cors.js` and `lisk_request_limit` use Chai to set up `sinon-chai`, which is redundant given our new setup.